### ISSUE TITLE
Unlock Convert-UrnToPath for ServerGroup/RegisteredServers on PS7

### ISSUE THREAD

**Issue by Matteo-T** [microsoft/SQLServerPSModule#40]
1. On PowerShell7, import v22.0.59
2. Run the following script
    ```pwsh
     Convert-UrnToPath 'RegisteredServersStore/ServerGroup[@Name=''DatabaseEngineServerGroup'']/ServerGroup[@Name=''G1'']'
     ```
3. Result
    ```console
    Operation is not supported on this platform.
    ```

**Comment by Matteo-T** [comment]
This will be fixed in any post v22.0.59 release.

**Comment by Matteo-T** [comment]
This was fixed in v22.1.1
